File: app/src/main/java/com/mithilakshar/mithilapanchang/eclipsedatamodel.java
package com.mithilakshar.mithilapanchang;

public class eclipsedatamodel {

    String eclipseName,eclipseDesc,imageUrl;

    public eclipsedatamodel() {

    }

    public eclipsedatamodel(String eclipseName, String eclipseDesc, String imageUrl) {
        this.eclipseName = eclipseName;
        this.eclipseDesc = eclipseDesc;
        this.imageUrl = imageUrl;
    }

    public String getEclipseName() {
        return eclipseName;
    }

    public void setEclipseName(String eclipseName) {
        this.eclipseName = eclipseName;
    }

    public String getEclipseDesc() {
        return eclipseDesc;
    }

    public void setEclipseDesc(String eclipseDesc) {
        this.eclipseDesc = eclipseDesc;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/com/mithilakshar/mithilapanchang/eclipsedatamodel.kt
package com.mithilakshar.mithilapanchang

class eclipsedatamodel {
    var eclipseName: String? = null
    var eclipseDesc: String? = null
    var imageUrl: String? = null

    constructor()
    constructor(eclipseName: String?, eclipseDesc: String?, imageUrl: String?) {
        this.eclipseName = eclipseName
        this.eclipseDesc = eclipseDesc
        this.imageUrl = imageUrl
    }
}
